recipes/dynare_preprocessor/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: dynare-preprocessor
+  version: 0.0.0.20231103.2a63840eb534c3dcd1f50477d935ecc92294f00b
+
+source:
+  url: https://git.dynare.org/Dynare/preprocessor/-/archive/6235e6907526d468e2c35f14c38db1e308698bc9/preprocessor.tar.gz
+  sha256: 06f76245ff5c6ac0811438cea6b817a321dc702c188b680e2a3191b54837c4ab
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - '{{ compiler("cxx") }}'
+    - meson
+    - bison
+    - flex
+    - pkg-config
+  host:
+    - gsl
+    - openlibm
```

Comment:
Not sure. Maybe it is needed by other parts of dynare other than the preprocessor. @svillemot ?

Reply:
I don't see anywhere in the logs that the build searches for or uses gsl. The linker logs show that sysroot libm is being used instead of openlibm, so if you want to use openlibm, you'll need to tweak the build to find the correct libm.

Reply:
Ok, then I'll try to remove it.

Reply:
removed

Reply:
`gsl` and `openlibm` are not needed and can be dropped. `boost` is used, but only its headers, not the shared libraries.